core/src/main/java/me/aurium/beetle/api/task/TaskFuture.java

```diff
@@ -0,0 +1,6 @@
+package me.aurium.beetle.api.task;
+
+import java.util.concurrent.CompletableFuture;
+
+public class TaskFuture<T> extends CompletableFuture<T> {
```

Comment:
Does this class add anything to CompletableFuture? If not, simply use a CompletableFuture

Reply:
This hasn't been worked on yet.